packages/server/src/api/controllers/row/ExternalRequest.ts

```diff
@@ -101,6 +104,26 @@ function buildFilters(
   }
 }
 
+function removeRelationships(
+  rowId: string,
+  table: Table,
+  isManyToMany: boolean,
+  colName?: string
+) {
+  const tableId = table._id!
+  const filters = buildFilters(rowId, {}, table)
+  // safety check, if there are no filters on deletion bad things happen
+  if (Object.keys(filters).length !== 0) {
```

Comment:
I've come across this in my review without any context on how it's used, but it strikes me as strange that this function just doesn't do anything under some circumstances. Should it throw an error or some such to let the caller know it didn't run?

Reply:
I agree this is a bit weird - but don't really want to make a change here as part of this PR - throwing an error could be risky, I believe its valid right now for this function to do nothing if there is no relationships to remove.